packages/svelte/src/remix-router-svelte.ts

```diff
@@ -110,6 +114,13 @@ export function useFetcher<TData = unknown>(): Readable<
     fetcherStore.set(router.getFetcher<TData>(fetcherKey));
   });
 
+  class FetcherForm extends Form {
```

Comment:
This is slick - components in Svelte are just classes you can extend?

Reply:
Yeah! After compilation, it is a plain JavaScript class. Extending them like this isn't super common, but it is available if you need it.